assets/js/base/components/cart-checkout/address-form/address-form.js

```diff
@@ -74,10 +77,21 @@ const AddressForm = ( {
 } ) => {
 	const {
 		getValidationError,
+		getValidationErrorId,
 		setValidationErrors,
 		clearValidationError,
+		hideValidationError,
 	} = useValidationContext();
 
+	const textInputValidationFunctions = {
+		getValidationError,
+		getValidationErrorId,
+		setValidationErrors,
+		clearValidationError,
+		hideValidationError,
+		validationInputError: ValidationInputError,
```

Comment:
Looks like ValidationInputError is a component so I would not pass it here along with these functions. From what I see it's usually passed as `inputErrorComponent`:
	
```
	<InlineCard
			onChange={ onChange }
			inputErrorComponent={ ValidationInputError }
		/>
```

Reply:
😍 Excellent spot and suggestion, thanks!